### DISCUSSION TITLE
USB C hub with dual hdmi monitors

### DISCUSSION THREAD

**Discussion by mg1234567** [emoacht/Monitorian#474] [Q&A]
I have a usb c hub 
Syiadx MST Dual HDMi Hub USB C to HDTV 12 Ports Multifunction Adapter 
There are a couple versions of this, if you want to know, the one I have has pd rj45 HDMI HDMI VGA on one side 

Using a laptop, I have 2 external monitors attached via HDMI 
Both show up in the icon as 
LG Full HD (32MN500M) 
and 
MX279  

Both say (in red) 
"Monitor is not controllable
DDC/CI is not supported or enabled"  

I'm not 100% the first is compatible, but the 2nd is and it's enabled (on) in the monitor's settings. 

Is there anything I can do? 
Is this an issue because of the hub or the monitors or ... 

Any help or info would be greatly appreciated. Thanks

**Comment by emoacht** [comment]
I guess that the hub is not compatible with DDC/CI. If it is the case, perhaps nothing you can do.
